pages/documentation/simulation/flowdecomposition/index.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: default
+latex: true
+---
+
+# Flow decomposition
+
+* TOC
+{:toc}
+
+## Introduction
+
+Flow decomposition methodology has been formerly described in a decision [published by ACER](https://eepublicdownloads.entsoe.eu/clean-documents/nc-tasks/CORE%20-%2074%20-%20ACER%20decision%20-%20Annex%201.pdf) (European Union Agency for the Cooperation of Energy Regulators).
+
+European power system is based on zonal management. Flow decomposition is a tool designed to give insights on
+impacts of internal and cross zonal exchange of power on the flows (and associated constraints) on the network devices.
+It is an important part of the cost sharing methodolgy for remedial actions costs sharing between TSOs.
+
+The aim of flow decomposition algorithm is to provide for each network element a decomposition of the active flow into different parts:
+- Allocated flow: flow due to electricity market exchanges. This includes import/export flows and transit flows.
+- Internal flow: flow due to electricity exchange inside the network element's zone.
+- Loop flow: flow due to electricity exchange inside another zone.
+- PST flow: flow due to a shift commanded by the action of an active phase shifting transformer on the network.
+- HVDC flow: flow due to a shift commanded by the action of an active HVDC line on the network.
+
+> Current algorithm does not model HVDC flow yet.
+
+> This decomposition does not reflects the exact reality of how electric flows act on a real network, but it
+> is a useful approximation needed for some cross zonal coordination processes.
+
+## Algorithm description
+
+The flow decomposition algorithm is based on the DC approximation, in which the losses in the network branches
+are neglected, and that allows to rely on the superposition principle to assess which is the impact of any injection
+on any branch flow by simple sensitivity analysis.
```

Comment:
on -> or ?

Reply:
No, it is the impact of an injection *on* branch flow (or i miss something ?). I.E. sensitivity matrix